packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -262,7 +262,7 @@ module.exports = (themeOptions = {}) => {
               query: `
               {
                 allReleaseNotePage(
-                  limit: 5, sort: { order: DESC, fields: date },
+                  limit: 10, sort: { order: DESC, fields: date },
```

Comment:
Actually, the idea was to show more release notes in the table, but i'm not sure if we really need more release notes. I think the list is long enough...at the moment there are 9 entries and in "worst case", there would be only 5 release notes shown in the table, so I think we just have to decide if we want to have a longer table.
@nkuehn what do you think?

Reply:
Yes, this is correct. The number must match the maximum number we want to show in the feed because worst case all 10 last releases to be shown in the aggregated table are from one specific feed so all feeds must provide the maximum number of entries that the aggregated list would show.